ServerCore/Pages/Resources/nipd2019/RulesPartial.cshtml

```diff
@@ -0,0 +1,93 @@
+        <li>You can use any reference source on the internet. You can use friends or strangers as reference sources but not to help you solve puzzles (e.g., you could call someone to ask them a trivia question but not send them a whole list of trivia questions for them to research). You may not send entire puzzles to friends for help.</li>
+        <li>
+            Writing a computer program to solve a puzzle is allowed, but this may take longer than solving the puzzle in its intended manner.
+        </li>
+        <li>
+            Hacking puzzles is not in the spirit of the event and is strictly prohibited. This includes:
+            <ul>
+                <li>hacking into any computer system to find answers</li>
+                <li>modifying something needed by other teams to solve any puzzle (e.g., moving an object referenced in a puzzle)</li>
+                <li>giving false information to another team</li>
+                <li>trying to purloin answers from another team, and</li>
+                <li>colluding with other teams</li>
+            </ul>
+        </li>
+        <li>In order for a puzzle to be marked as solved, the correct answer must be entered into the correct answer submission box on the website and be submitted before the event ends.</li>
+        <li>Each team must be reachable by telephone and email during the event. New puzzles and corrections to puzzles will be sent out via email. <b>Ensure that you are set up to receive email from msph@puzzlehunt.org.</b></li>
+        <li>
+            Teams should clean up their conference rooms after the end of the event.
+            The regular cleaning schedule does not cover weekends. You should leave the room in an appropriate condition to be used for a Monday morning meeting.
+        </li>
+    </ol>
+    <h3>Traveling outside your conference room</h3>
+    <ol>
+        <li>There are ONLY two ways that you might be instructed to travel.</li>
+        <ul>
+            <li>The puzzle has explicit instructions to travel. For example, your team may be scheduled for a live event at a specified location.</li>
+            <li>You submit an answer and the response from the website gives you explicit instructions to travel. If you have an answer that looks like a travel instruction, enter it into the submission box in full and only travel if the response says so.</li>
+        </ul>
+        <li>
+            You must not enter any person's private office during the course of the event unless it is the office of a member of your team.
+        </li>
+        <li>
+            If any travel is required, it will take place East of SR-520, South of NE 40th Street, and North of Bel-Red Road. Do not enter the following buildings unless your personal office is in them: 26, 30, 34, 50, and 87. Do not enter any construction areas.
```

Comment:
Do we have an up to date list of buildings that includes West campus?

Reply:
Nope, not going to bother since this statement is technically true as we know it.